saleor/graphql/product/filters.py

```diff
@@ -185,7 +185,13 @@ def _clean_product_attributes_date_time_range_filter_input(filter_value):
                     gte, datetime.datetime.min.time(), tzinfo=pytz.UTC
                 )
             filters["date_time__gte"] = gte
-    return matching_attributes.filter(**filters)
+    matching_queryset = matching_attribute_values.filter(**filters)
+    if not matching_queryset:
+        raise ValueError(
+            f"Requested value for attributes {attribute_slugs} doesn't exist"
+        )
+    for attr_val in matching_queryset:
+        queries[attr_val.attribute.pk] += [attr_val.pk]
```

Comment:
This means additional DB query for fetching `attribute` for each `attr_val`

Reply:
Also keep in mind, that these changes were previously refactored because they were slow. It was done here: https://github.com/saleor/saleor/pull/10199
Have you tested your implementation on a bigger dataset?